### PR TITLE
Connect technologies with new backend model

### PR DESCRIPTION
**Pull request by Czaroos**
changed avatar src for technologies to be pictureUrl from backend. Also: if other extensions than .svg will be allowed - some additional styling is needed or maybe we should use Avatar from material UI instead of img tag?

### CODE REVIEW

File: src/ui/select-base/SelectBase.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ namespace SelectBase {
     dataIdx: string;
     label: string;
     value: boolean;
+    pictureUrl?: string;
```
Comment (polubis) [inline src/ui/select-base/SelectBase.tsx:34]: Remove pictureUrl. SelectBase don't care about other properties.

**Review by polubis — CHANGES_REQUESTED** [review]

File: src/ui/select-base/SelectBase.tsx (hunk 1 above)
Comment (Czaroos) [reply src/ui/select-base/SelectBase.tsx]: Ok, it's been removed - therefore implementation of getting pictureUrl must have been changed. Check this out and give me a feedback if this approach is acceptable

**Review by polubis — CHANGES_REQUESTED** [review]
I added commit - with mapping method removal. All that you need is just smart type definitions. 

Also, I checked how technologies behave in some places. There is needed some work to do with UI but this will be done after Database clear - because right now we have some invalid technologies mocked.

Just check my commit - and this task is done right now.

**Review by polubis — APPROVED** [review]

**Review by polubis — APPROVED** [review]
Review done - will be merged in feature branch

**Pull request closed without merging**